reference/modules/telemetry/specialized.md

```diff
@@ -0,0 +1,283 @@
+
+:::{table}
+:align: center
+| Index in settings list | The planet that the setting turnes on/off |
+|------------------------|-------------------------------------------|
+| 0                      | Mercury                                   |
+| 1                      | Venus                                     |
+| 2                      | Earth                                     |
+| 3                      | Mars                                      |
+| 4                      | Jupiter                                   |
+| 5                      | Saturn                                    |
+| 6                      | Uranus                                    |
+| 7                      | Neptune                                   |
+:::
+
+Here is an example of how a message from this telemetry type can look:
+:::{code-block}
+[ /Overview, Int8Array[ 0, 0, 1, 0, 1, 1, 1, 1 ] ]
+:::
+
+Here is a breakdown of the example message above:
+- `/Overview`: The OSC label indicating that this is a message from the **Planets Overview Sonification** telemetry type.
+- `Int8Array[ 0, 0, 1, 0, 1, 1, 1, 1 ]`: The list of user interface settings, where:
+    - `0` Mercury, is turned off.
+    - `0` Venus, is turned off.
+    - `1` Earth, is turned on.
+    - `0` Mars, is turned off.
+    - `1` Jupiter, is turned on.
+    - `1` Saturn, is turned on.
+    - `1` Uranus, is turned on.
+    - `1` Neptune, is turned on.
+
+# Sonification
+There is a sonification of the planets in our solar system that can be used together with the telemetry module. Sonification is when information is conveyed using sound in contrast to visualization when images are used. The files for the sonification can be found in _data\assets\modules\telemetry\sonification_ in the OpenSpace folder. The file _planets.asset_ is a regular Lua script OpenSpace asset that configures the telemetry module to monitor each of the planets in the solar system and their major moons. The sonification file itelf _OpenSpaceSonification.scd_ is a SuperCollider file and needs to be run seperatly in that program. The sonification will then be produced by SuperCollider and just uses the informaiton that OpenSpace sends with the OSC messages provided by the telemetry module.
+
+:::{note}
+Note that the _planets.asset_ file cannot be customized or altered since the sonification expects the data from the telemetry module to be in a certain format determined by this file. If this file is altered, then the sonification file needs to be updated to reflect the change as well. If you want to create your own sonification for other objects in OpenSpace other than the planets, then the general [Customized Nodes Information](./general.md#customized-nodes-information) telemetry type is likely a better fit for that purpose.
```

Comment:
```suggestion
Note that the _planets.asset_ file cannot be customized or altered since the sonification expects the data from the telemetry module to be in a certain format determined by this file. If this file is altered, then the sonification file needs to be updated to reflect the change as well. If you want to create your own sonification for objects in OpenSpace other than the planets, then the general [Customized Nodes Information](./general.md#customized-nodes-information) telemetry type is likely a better fit for that purpose.
```